manifests/latest/vsphere-7.0/supervisorcluster/k8s-1.15/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -35,8 +35,6 @@ spec:
             - "--v=4"
             - "--timeout=300s"
             - "--csi-address=$(ADDRESS)"
-            - "--feature-gates=Topology=true"
```

Comment:
WCP needs volume topology feature for host local provisioning.

Reply:
I see the code - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/wcp/controller.go#L228-L229

Let me revert back then for WCP.

Reply:
pushed commit (https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/381/commits/5976e85be6ae2cde90eb2cede8633604b23baf41) to change on vanilla YAML files.